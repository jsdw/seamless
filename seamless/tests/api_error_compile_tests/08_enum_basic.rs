// FAIL: attrs that we need weren't provided.

#[derive(seamless::ApiError)]
enum Foo {
    Bar,
    Wibble
}

fn main() { }
