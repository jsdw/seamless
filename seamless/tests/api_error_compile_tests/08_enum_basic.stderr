error: One of '#[api_error(internal)]' or '#[api_error(external)]' or '#[api_error(external = "foo")]' is required (1)
 --> $DIR/08_enum_basic.rs:5:5
  |
5 |     Bar,
  |     ^^^
